src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+    <form 
+      class="form__big"
```

Comment:
by BEM it should be `form form--big` (block + modifier), not `from__big` (element), bc then you haven't `form` block that you use for elements lower